astropy/modeling/fitting.py

```diff
@@ -1218,14 +1218,17 @@ def _wrap_deriv(params, model, weights, x, y, z=None):
                         output = np.array(
                             [np.ravel(_) for _ in np.atleast_2d(weights).T * fit_deriv]
```

Comment:
@WilliamJamieson, for a separate discussion, [`numpy.transpose()` docs](https://numpy.org/doc/stable/reference/generated/numpy.transpose.html) says 

```suggestion
                            [np.ravel(_) for _ in np.transpose(weights) * fit_deriv]
```

should be identical